src/client/components/share-button/readme.md

```diff
@@ -0,0 +1,23 @@
+# Share Button
+
+### Features:
+
+- Supports native share
+- Supports share to Twitter
+- Supports share to Facebook
+- Supports share to Linkedin
+- Supports share to WhatsApp
+- Supports share via e-mail
+- Supports copy to clipboard*
+
+- Tracks social interactions using Google Analytics
+
+* This feature doesn't work on iPhone/iPad, so when the user clicks on it, an error message is displayed.
```

Comment:
using * like this makes it another list item in Markdown, which makes it unclear that _this feature_ is related to copy to clipboard. So you can either remove the spacing after * or add it in parentheses after the _supports copy to clipboard_ item

Reply:
..and please rephrase to say this button is not displayed on iPhone/iPad for that reason.